index.js

```diff
@@ -0,0 +1,18 @@
+const prompt = require('prompt-sync')();
+
+// program to generate fibonacci series up to n terms
+
+// take input from the user
```

Comment:
We should avoid such comments as it is 100% clear from the line below.

The goal is to explain what code does not explain - often the why part.

Please do a quick google to read up on what makes up for good v/s not so good comments.

Reply:
Ex: https://itnext.io/what-makes-a-good-code-comment-5267debd2c24

Reply:
https://news.ycombinator.com/item?id=18157047
https://news.ycombinator.com/item?id=23749676
https://news.ycombinator.com/item?id=19313562

Reply:
I went through the blogs. Explaining what does the code do is a bad comment as it does make sense.